perf/benchmark/runner/runner.py

```diff
@@ -152,16 +152,11 @@ def run(self, conn, qps, size, duration):
         if duration is None:
             duration = self.duration
 
-        labels = self.run_id
-        labels += "_qps_" + str(qps)
-        labels += "_c_" + str(conn)
-        labels += "_" + str(size)
```

Comment:
This part is how the labels in the csv file being generated from. Like what you saw: `8d5f4095_qps_1000_c_4_1024_nomixer_both`

Reply:
Yes, I removed the labels because I didn't see that they where used somewhere.